src/graphql/graphql.schema.ts

```diff
@@ -140,99 +162,69 @@ const schema = gql`
     hasLimitedResults: Boolean
   }
 
-  type MetaInformation {
-    description: String
-    themes: [Float]
-    datasetIds: [Float]
-    thumbnail: String
-    date: String
+  type MapCollection {
+    id: ID!
+    title: String!
+    mapLayers: [MapLayer!]!
+    meta: Meta!
   }
 
-  interface MapLayer {
-    id: String
-    title: String
-    layers: [String]
+  type MapLayer {
+    id: ID!
+    title: String!
+    type: String!
     url: String
-    params: String
```

Comment:
This seems to be missing, but is required in the `atlas` application

Reply:
Good catch, I've added some of the fields that were missing.